pkg/kube/kube_handler.go

```diff
@@ -312,7 +307,6 @@ type handler interface {
 	UpdateAppRepository(appRepoBody io.ReadCloser, requestNamespace string) (*v1alpha1.AppRepository, error)
 	RefreshAppRepository(repoName string, requestNamespace string) (*v1alpha1.AppRepository, error)
 	DeleteAppRepository(name, namespace string) error
```

Comment:
just curious: do you know if we can remove more logic from this file? If so, do you plan to do so as part of the kubeops deprecation? Or should we file another issue for it?

Reply:
Exactly, from the `pkg/kube` we can remove a lot of stuff. That will be done after removing Kubeops in a subsequent step.